controllers/handlerFactory.js

```diff
@@ -17,7 +17,10 @@ exports.createOne = Model =>
       };
     });
     const image = await Image.create(images);
-    const resources = await Resources.create({ images: image });
+    const resources = await Resources.create({});
+    image.forEach(image => resources.images.push(image._id));
```

Comment:
why don't you create the images first and then create the resources with its corresponding images? so instead of two db requests it becomes one request

Reply:
we save images to a separate collection, and resources to a different one, then we link the created (1 or 2 ) images to the created resources document to be a wrapper for the created images and finally link the resources to the created Poll.

if you have any hints to optimize it, go on I'm curious to work on it.

Reply:
is this solution acceptable?
[one DB request commit](https://github.com/m3ntorship/pickly-backend/pull/6/commits/d49693dd517c518280347ee3a65415655d93a8c2)

Reply:
remove the `await` from line 20

Reply:
yes, this is what I meant by comment.